rtengine/dcraw.cc

```diff
@@ -6550,8 +6550,6 @@ int CLASS parse_tiff_ifd (int base)
 	break;
       case 324:				/* TileOffsets */
 	tiff_ifd[ifd].offset = len > 1 ? ftell(ifp) : get4();
-	if (len == 1)
-	  tiff_ifd[ifd].tile_width = tiff_ifd[ifd].tile_length = 0;
```

Comment:
Removing this will affect the logic here:
https://github.com/Beep6581/RawTherapee/blob/2ce5b827c5fa39b2b5425303f79f28b78f3df4f4/rtengine/dcraw.cc#L7065-L7066
...but I'm not sure why you would like to set `tile_width`/`tile_length` to 0 or INT_MAX...

Reply:
Seems like this breaks lossless JPEG decoding for some reason...

Reply:
Removing
https://github.com/Beep6581/RawTherapee/blob/2ce5b827c5fa39b2b5425303f79f28b78f3df4f4/rtengine/dcraw.cc#L1136-L1137
seems to solve the problem for lossless JPEG encoded single tiled DNGs, but will at the same break things for some multi-tiled images...

It might also be worth looking into the logic deciding between the two functions `lossless_jpeg_load_raw` and `lossless_dng_load_raw`.